src/controllers/comment.js

```diff
@@ -0,0 +1,51 @@
+import { sendDataResponse } from '../utils/responses.js'
+import ERR from '../utils/errors.js'
+import {
+  createCommentDb,
+  getAllCommentsDb,
+  getAllPostCommentsDb,
+  postFound
+} from '../domain/comment.js'
+
+const getAllComments = async (req, res) => {
+  const comments = await getAllCommentsDb()
+
+  return sendDataResponse(res, 200, { comments })
+}
+
+const createComment = async (req, res) => {
+  const { content } = req.body
+
+  if (!content) {
+    return sendDataResponse(res, 400, { error: ERR.MISSING_CONTENT })
+  }
+
+  const { id } = req.user
+  const postId = Number(req.params.id)
+
+  const post = await postFound(postId)
+
+  if (!post) {
+    return sendDataResponse(res, 400, { error: ERR.POST_NOT_FOUND })
+  }
+
+  const comment = await createCommentDb(content, id, postId)
```

Comment:
If the requirements ask for a 255 character limit, you need to validate the data before trying to save it to the DB